### DISCUSSION TITLE
DJRemix

### DISCUSSION THREAD

**Discussion by Proelsaprol** [zkSync-Community-Hub/zksync-developers#371] [Dev Tooling]
// SPDX-License-Identifier: MIT
pragma solidity ^0.8.0;

contract DJRemix {
    // The address of the DJ
    address private djAddress;

    // Constructor to store the DJ's address when the contract is deployed
    constructor() {
        djAddress = address(0x96438438073E41271f5fE6395B7026311E48DA74);
    }

    // Method to get the shoutout, returning the DJ's address
    function getShoutout() public view returns (address) {
        return djAddress;
    }
}

**Comment by dutterbutter** [comment]
This is not a question. Please re-open with a question.
